MetaMask/metamask-extension
# Created a custom spending cap component

## Explanation

Created a custom spending cup component. This component can't yet be added to the app. Instead, I added a storybook story that displays this component. This component received at least the following props:

- tokenName (type: string)
- currentTokenBalance (type: number)
- dappProposedValue (type: number)
- siteOrigin (type: string)
- onEdit (type: function)

## More Information

* Fixes #15224 

## Screenshots/Screencaps

https://user-images.githubusercontent.com/92527393/192730850-1d116e39-e289-4682-9cf3-f6ee450c47ae.mov

## Manual Testing Steps

This component can be tested by running storybook (`yarn storybook`) and then go to the `COMPONENTS` tab and open in `App/CustomSpendingCup`. For testing this component you can see the Acceptance Criteria in the ticket: #15224.
